presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -2656,12 +2656,9 @@ core::ExecutionStrategy toStrategy(protocol::StageExecutionStrategy strategy) {
         FIXED_LIFESPAN_SCHEDULE_GROUPED_EXECUTION:
     case protocol::StageExecutionStrategy::
         DYNAMIC_LIFESPAN_SCHEDULE_GROUPED_EXECUTION:
+    case protocol::StageExecutionStrategy
```

Comment:
Would you add a comment here to explain why we don't need to distinguish between RECOVERABLE_GROUPED_EXECUTION, DYNAMIC_LIFESPAN_SCHEDULE_GROUPED_EXECUTION and FIXED_LIFESPAN_SCHEDULE_GROUPED_EXECUTION?